components/admin_console/license_settings/starter_edition/starter_left_panel.tsx

```diff
@@ -73,14 +75,21 @@ const StarterLeftPanel: React.FC<StarterEditionProps> = ({
                 <div className='uploadButtons'>
                     <button
                         className='btn btn-upload light-blue-btn'
-                        onClick={openUploadModal}
+                        onClick={() => fileInputRef.current !== null && fileInputRef.current.click()}
```

Comment:
What about `() => fileInputRef.current?.click()` ?

Reply:
typescript linter was complained with that approach